src/ledger/LedgerTxn.cpp

```diff
@@ -2397,7 +2397,8 @@ LedgerTxnRoot::Impl::getBestOffer(Asset const& buying, Asset const& selling,
         prefetch(toPrefetch);
 
         cached->allLoaded =
-            std::distance(newOfferIter, offers.cend()) < BATCH_SIZE;
+            static_cast<size_t>(std::distance(newOfferIter, offers.cend())) <
```

Comment:
Exactly the same thing appears on line 2310. Is there no warning there?

Reply:
interestingly it wasn't... regardless, I fixed the other call site to be consistent